docs/api/plots/scatter.en.md

```diff
@@ -54,13 +54,13 @@ scatterPlot.render();
 
 <description>**optional** _jitter | stack | symmetric | dodge_ _default:_ `jitter`</description>
 
-数据调整类型，不建议修改。
+Adjust the type. Modification is not recommended.
```

Comment:
Adjust the data. 
Adjust types provided by G2Plot includes `'stack'`, `'dodge'` `'jitter'`, `'symmetric'`. Not recommended to modify.